### PR TITLE
Fix ocata bundle

### PR DESCRIPTION
**Pull request by andrewdmcleod**
Details in commit messages - remove cpu-mode, specify ext-port.

### CODE REVIEW

File: bundle-ocata-s390x.yaml [hunk 1]
```diff
@@ -190,7 +190,7 @@ services:
     num_units: 1
     options:
       worker-multiplier: 0.5
-      bridge-mappings: physnet1:br-ex
+      ext-port: eth1
```
Comment (ryan-beisner) [inline bundle-ocata-s390x.yaml:193]: We should keep the new-style network declaration (physnet).  Is there a bug causing the need to change this to the old-style network declaration?

Comment (andrewdmcleod) [reply bundle-ocata-s390x.yaml:193]: No reason other than what works in the mitaka bundle is ext-port. Will retry with physnet:eth1

Comment (andrewdmcleod) [reply bundle-ocata-s390x.yaml:193]: Rather:

bridge-mappings:         physnet1:br-ex
    data-port:               br-ex:eth1

Comment (ryan-beisner) [reply bundle-ocata-s390x.yaml:193]: Ack. I've also not been able to make the new-style network expression work with the openstack-on-lxd scenarios.

@javacruft - do we need to keep using the old-style ext-port approach for this scenario?  Can you think of any reasons the new style wouldn't work?

**Pull request merged**